packages/mermaid/src/Diagram.ts

```diff
@@ -81,6 +82,8 @@ export const getDiagramFromText = async (
   text: string,
   metadata: Pick<DiagramMetadata, 'title'> = {}
 ): Promise<Diagram> => {
+  text = encodeEntities(text);
+
```

Comment:
The right place for this would be inside the constructor of `mermaid/packages/mermaid/src/Diagram.ts`.

Reply:
Rethinking, I feel it should be in utils along with `decodeEntities` reason being these are pure JS functions and reusable as well so utils looks like a good place and these can be separately unit tested as well. Let me know if this sounds good.

Reply:
I meant where the function is to be called, not the implementation.